framework/PluginsBaseClasses/PlotPluginBase.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Provides API and utilities for extending the OutStream Plot with custom plotting options.
+Created on March 11, 2021
+@author: talbpaul
+"""
+from __future__ import division, print_function , unicode_literals, absolute_import
```

Comment:
This is not needed for python3, right?

Reply:
I think it may be worth to move this to a separate pull request.

Reply:
I did not mean to include this! Thanks for noting it.